mailpoet/assets/js/src/automation/editor/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './panel';
+export { PanelBody } from '@wordpress/components';
```

Comment:
Seeing premium ([here](https://github.com/mailpoet/mailpoet-premium/pull/665/commits/2fccce5dddd134005893f94af8a71bc7ff56ebbe)), I think this should be exported directly in `webpack_admin_expose.js` as [the other components are](https://github.com/mailpoet/mailpoet/blob/3279fb1154a98f19b5b09d925cd24d8159bc7723/mailpoet/assets/js/src/webpack_admin_expose.js#L13-L15).

Then, in Premium, it should be imported as `@wordpress/components/panel-body`, not under automation, I think.

Reply:
[6a4d732](https://github.com/mailpoet/mailpoet/pull/4604/commits/6a4d73232b2d5cd4a5aeaac743cb26ebb8726bf0)